setup.py

```diff
@@ -15,7 +15,7 @@
     'dask[complete] >=0.18.0',
     'toolz >=0.7.4',  # ? for some dask issue (dasks does only >=0.7.3)
     'datashape >=0.5.1',
-    'numba >=0.37.0,<0.49',
+    'numba <0.49.0,>=0.51.0',
```

Comment:
This change now claims that Datashader can work with Numba older than 0.37.  Is that really true?  I don't recall any patches added for working with older Numba versions like that.  I think we need to express that it works with Numba greater than 0.37 _except_ 0.49 and 0.50; not sure how to say that in this expression language.

Reply:
That's true we should keep the lower pin but is `'numba >=0.37.0,<0.49.0,>=0.51.0'` even valid?

Reply:
I think in that case, something like this would work: `numba >=0.37.0, <0.49.0,>=0.51.0`

Reply:
@philippjfr `numba >=0.37.0,<0.49.0,>=0.51.0` this will not resolve as all 3 conditions will never be satisfied. This seems to work on my end: `'numba >=0.37.0, !=0.49.*, !=0.50.*'`

Reply:
Right, that was my worry and thanks for figuring that out. Excluding 0.49 and 0.50 like that seems like the best option.

Reply:
Sounds good, thanks!

Reply:
fixed it